### ISSUE TITLE
placement not working right

### ISSUE THREAD

**Issue by patrick-visniewski** [Ben-Ramchandani/OutpostPlanner#23]
Not sure if this issues gets used, I posted this on the forum for the mod but I'll put it here too

This is happening since I updated to 0.18.41 or maybe as early a 0.18.39 
I set a simple bot based blue print for the blue print. It has 2 drills facing each other one space apart with a chest between then.
And it has a medium power pole next to the chest.
I have smart placement on as well for power poles.

The results is that 
1) few or not chests gets placed, they only get placed if there is no ore for the drill facing a drill to be placed (and no, there really is space for the chest between all placed drills)
2) drills get spaced out with 1 space separating neighboring drills
3) only 1 power pole gets placed
4) no errors appear other than the message that it could not place poles. there are no errors in the log either

I hope the image file is available. it shows the results now with the spacing, no chests and just 1 pole.

![outpost_planner_issue](https://user-images.githubusercontent.com/28932133/89233322-f7677600-d5ae-11ea-9f6e-7fedf9d2bd7e.png)

**Comment by danielnichols** [comment]
I'm also having this issue, but with the default belt-based blueprint. Factorio 1.0.0 and Outpost Planner 1.1.5

**Comment by patrick-visniewski** [comment]
yeah as far as I know they changed something in Factorio having to do with blueprint placement or... not sure what

But Picker Dollies had a similar issue. It was moving items 2 space rather than 1 each time after the Factorio updates. 
The author of that fixed that mod, and I checked the changes but I don't see how I'd do something similar with code for this mod or probably the planner core mod really.

Unfortunately I don't understand the cause well enough to try to fix the code myself ... yet.
But using the other methods to figure out the size or maybe collision boundaries report different values now than they used to.
